CMakeLists.txt

```diff
@@ -15,4 +14,6 @@ target_include_directories(${PROJECT_NAME} INTERFACE
 target_compile_features(${PROJECT_NAME} INTERFACE cxx_std_11)
 
 include(InstallTarget)
-install(DIRECTORY ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
\ No newline at end of file
+install(DIRECTORY ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+add_subdirectory(tests)
```

Comment:
I used to have a specific build job for tests so that if anyone ever installs the lib, they won't need to install whatever test framework I use.
No idea if it's a good practice or not.

Reply:
`With your method, I like the fact that it's testing the install space.`
Yep, that was the plan, now that I remember it.

I'd rather keep it. Since I bet no one will ever want to run my tests anyways.